app/models/event.rb

```diff
@@ -28,6 +28,8 @@ class Event < ActiveRecord::Base
   has_many :participant_groups
   has_many :date_ranges
   accepts_nested_attributes_for :date_ranges
+  validates_presence_of :name
+  validates_presence_of :description
```

Comment:
Nah, I like it, This way I got to refator all `validate :xyz, presence: true` to this more readable form (at least in my eyes).